commit 5725b4b411340548e5340be020f28b7532900438 (refs/changes/57/1257/2)
Author: David Pacheco <dap@joyent.com>
Date:   2017-01-13T17:20:14-08:00 (2 years, 9 months ago)
    
    joyent/node-manta#298 mjob-simple fails because of GNU date regression
    Reviewed by: Brian Bennett <brian.bennett@joyent.com>
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Brian Bennett <brian.bennett@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index f646ced..7164278 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -4,6 +4,7 @@
 
 - joyent/node-manta#230 Allow redirection of `mlogin` debug log output on
   `stderr`; e.g., `LOG_LEVEL=trace mlogin -v 2> >(bunyan -o short)`.
+- joyent/node-manta#298 mjob-simple fails because of GNU date regression
 
 ## 4.1.1
 
diff --git a/test/mjob-simple.test.js b/test/mjob-simple.test.js
index bcd6cef..e236a82 100644
--- a/test/mjob-simple.test.js
+++ b/test/mjob-simple.test.js
@@ -68,17 +68,15 @@ function mjobExec(args, opts, cb) {
 // ---- tests
 
 /*
- * Note the usage of '-or date' cuddled like that is ensuring we aren't
+ * Note the usage of '-or "echo hello"' cuddled like that is ensuring we aren't
  * hitting <https://github.com/trentm/node-dashdash/issues/8>.
  */
-test('mjob create --close -or date', function (t) {
-    var args = ['create', '--close', '-or', 'date'];
+test('mjob create --close -or "echo hello"', function (t) {
+    var args = ['create', '--close', '-or', 'echo hello'];
     mjobExec(args, function (err, stdout, stderr) {
         t.ifError(err, err);
         t.equal(stderr, '', 'stderr is empty: ' + stderr);
-        // Example: 'Fri Jun 24 19:05:33 UTC 2016\n'
-        t.ok(/^\w{3} \w{3} [ \d]{2} [ :\d]{8} UTC \d{4}\n$/.test(stdout),
-            'stdout is a date: ' + stdout);
+        t.equal(stdout, 'hello\n', 'stdout mismatch');
         t.done();
     });
 });
